docs/tutorials/mlflow.ipynb

```diff
@@ -34,7 +34,7 @@
    "outputs": [],
    "source": [
     "# Install MLRun and scikit-learn if not already installed. Run this only once. Restart the notebook after the install!\n",
-    "%pip install mlrun scikit-learn~=1.5.1 xgboost~=2.0.3"
+    "%pip install mlrun scikit-learn~=1.5.1 xgboost~=2.0.3 setuptools~=69.5.1"
```

Comment:
mlrun requires `setuptools>=75.2` so I don't think downgrading it is the best option.
is it possible to bump mlflow instead?

Reply:
tested and verified, thanks @alonmr !
did the same for 1.7 PR